packages/@o3r/third-party/testing/jest.config.it.js

```diff
@@ -0,0 +1,8 @@
+const { join } = require('node:path');
+const getJestConfig = require('../../../../jest.config.it').getJestConfig;
```

Comment:
```suggestion
const { getJestConfig } = require('../../../../jest.config.it');
```

to be consistent with the previous line :D

Reply:
let's align all packages in another PR :-) 
![image](https://github.com/user-attachments/assets/d7181b66-153b-485e-b888-16805c9f6559)